restapi/datasource_api_object.go

```diff
@@ -1,57 +1,58 @@
 package restapi
 
 import (
-	"github.com/hashicorp/terraform/helper/schema"
 	"log"
+
+	"github.com/hashicorp/terraform/helper/schema"
 )
 
-func dataSourceRestApi() *schema.Resource {
+func dataSourceRestAPI() *schema.Resource {
 	return &schema.Resource{
-		Read: dataSourceRestApiRead,
+		Read: dataSourceRestAPIRead,
 
 		Schema: map[string]*schema.Schema{
-			"path": &schema.Schema{
```

Comment:
What is the affect of changing this? I'm assuming nothing since the map is declared to be a set of schema pointers keyed on a string... is this just shorthand? I ask because the Terraform provider docs suggested this format so I just rolled with it.